### DISCUSSION TITLE
notes/%E6%BD%AE%E6%B1%95%E8%AF%9DTTS/

### DISCUSSION THREAD

**Discussion by giscus** [Jackiexiao/obsidian-github-publish#37] [General]
# notes/%E6%BD%AE%E6%B1%95%E8%AF%9DTTS/

jackiexiao's blog

https://blog.jackiexiao.com/notes/%E6%BD%AE%E6%B1%95%E8%AF%9DTTS/

**Comment by p1an-lin-jung** [comment]
楼主，我最近在做一个潮州话数据集，用于ASR、TTS和潮普间的语言翻译，有兴趣交流吗
